application/tests/api/MfaCest.php

```diff
@@ -59,6 +59,23 @@ public function test31(ApiTester $I)
         $I->seeResponseCodeIs(403);
     }
 
+    public function test33(ApiTester $I)
+    {
+        $I->wantTo('check response when making PUT request to mfa/{id}/webauthn/{webauthn_id} with incorrect token');
+        $I->haveHttpHeader('Authorization', 'Bearer invalidToken');
+        $I->sendPUT('/mfa/5/webauthn/6');
+        $I->seeResponseCodeIs(401);
+    }
+
+    public function test34(ApiTester $I)
+    {
+        $I->wantTo('check response when making PUT request to mfa/{id}/webauthn/{webauthn_id} for a user'
+            . ' with auth_type=reset');
+        $I->haveHttpHeader('Authorization', 'Bearer user5');
+        $I->sendPUT('/mfa/5/webauthn/6');
+        $I->seeResponseCodeIs(403);
```

Comment:
I think the RAML will need to be updated for that endpoint. It doesn't seem to include `403` as a possible response code for `PUT /mfa/{id}/webauthn/{webauthn_id}`.